MainApp/.gitignore

```diff
@@ -8,4 +8,4 @@ tmp/
 !.gitkeep
 dist
 .idea/
-public/downloads/*
```

Comment:
@adityab17190 Maybe you should create a new branch from master and cherry pick your own commits into it.